kinto/config/kinto.tpl

```diff
@@ -78,9 +159,80 @@ multiauth.policies = basicauth
 # fxa-oauth.relier.enabled = true
 # fxa-oauth.webapp.authorized_domains = *
 
+# Plugins
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#plugins
+# https://github.com/uralbash/awesome-pyramid
+kinto.includes = kinto.plugins.default_bucket
+#                kinto.plugins.admin
+#                kinto.plugins.accounts
+#                kinto.plugins.history
+#                kinto.plugins.quotas
+
+# Pluggable components
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#pluggable-components
 #
-# Client cache headers
+# kinto.logging_renderer = your_log_renderer.CustomRenderer
+
+# Notifications
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#notifications
+#
+# Configuration example:
+# kinto.event_listeners = redis
+# kinto.event_listeners.redis.use = kinto_redis.listeners
+# kinto.event_listeners.redis.url = redis://localhost:6379/0
+# kinto.event_listeners.redis.pool_size = 5
+# kinto.event_listeners.redis.listname = queue
 #
+# Setting up notifications via WebSocket
+# https://kinto.readthedocs.io/en/latest/tutorials/notifications-websockets.html#tutorial-notifications-websockets
+
+# Filtering
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#filtering
+#
+# It is possible to filter events by action and/or types of object.
+# By default actions create, update and delete are notified for every kinds of objects.
+# kinto.event_listeners.redis.actions = create
+# kinto.event_listeners.redis.resources = bucket collection
+
+# Cross Origin Requests
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#cross-origin-requests-cors
+#
+# kinto.cors_origins = *
+
+# Backoff indicators/end of service
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#backoff-indicators
+# https://kinto.readthedocs.io/en/latest/api/1.x/backoff.html#id1
+#
+# kinto.backoff = None
+# kinto.retry_after_seconds
+# kinto.eos = None
+# kinto.eos_message = None
+# kinto.eos_url = None
```

Comment:
We have to choose between: no value, empty string`''` or `None`, and use the same everywhere. (I have a preference for no value)